src/game/test/gameTimerTextBegin.test.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @jest-environment jsdom
+ */
+"use strict";
+const { jest } = require("@jest/globals");
+const { setTimer } = require("../gameTimer");
+
+const timer = document.createElement("div");
+timer.classList.add("timer");
+document.querySelector("body").append(timer);
+
+const mockFunction = jest.fn();
+
+describe("Using function setTimer", () => {
+  describe("parameteres provided", () => {
+    test.each`
+      input
+      ${{ minutes: 2, textContent: "02:00" }}
```

Comment:
Make 3 or 5 examples to test. What is the point of testing only one variant in which everything gonna is ok? It should work for any time, yup? What will happen if I will pas 128? 2048? 0.5? Test not only obvious but also corner cases.

Reply:
Addmore scenarios and test for number smaller than 1, bigger than 60 and inputs in other format